community/pmount/build

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+
+patch -p1 < 001-disable-intltool.patch
+patch -p1 < 001-fix-musl-build.patch
+
+# autoconf warnings are treated as errors.
+set +e
+automake
+autoconf
+set -e
+
+./configure \
+    --prefix=/usr \
+    --disable-nls \
+    --with-media-dir=/mnt/
+
+make
```

Comment:
Without running `automake && autoconf` and instead using `autoreconf -sfi`, you'd have to add
`sed -i 's/po src tests/src/' Makefile` before `make`.